R/colour-text.r

```diff
@@ -5,6 +5,7 @@ testthat_colours <- list(
   error = crayon::red
 )
 
+#' @export
 colourise <- function(text, as = c("passed", "skipped", "error")) {
```

Comment:
I think the parameters `fg` and `bg` should remain and trigger a warning when used.